Documentation/feature-band-source-building.md

```diff
@@ -0,0 +1,773 @@
+  - 3xx becomes unsupported when 4xx is released
+
+## Build Requirements by Feature Band
+
+Each feature band has specific requirements for SDKs and artifacts:
+
+### 1xx Band Build Requirements
+
+- **Bootstrap (any version)**: Two-stage process using Microsoft source-built
+  1xx artifacts + Microsoft SDK + script
+- **Servicing (N.0.101+)**: Source-built SDK and artifacts from the previous
+  1xx release
+
+Note: Only the 1xx SDK is guaranteed to build the shared runtime components
+
+### 2xx Band Build Requirements
+
+- **Bootstrap (any version)**: Two-stage process using Microsoft source-built
+  2xx artifacts + Microsoft 2xx SDK + prep script
+- **Initial Release (N.0.200)**: Current source-built 1xx artifacts + current
+  source-built 1xx SDK
+- **Servicing (N.0.201+)**: Source-built SDK and artifacts from the previous
+  2xx release + current 1xx artifacts
+
+### 3xx/4xx Band Build Requirements
+
+- **Bootstrap (any version)**: Two-stage process using Microsoft source-built
+  Bxx artifacts + Microsoft Bxx SDK + prep script
+- **Initial Release (N.0.B00)**: Current source-built 1xx artifacts + current
+  source-built (B-1)xx artifacts + current source-built (B-1)xx SDK
+- **Servicing (N.0.B01+)**: Source-built SDK and artifacts from the previous
+  Bxx release + current 1xx artifacts
+
+### Feature Band Key Points
+
+- Release schedules across bands are not necessarily aligned (unless there is
+  a security release in the shared components in which case they will be aligned).
+- 1xx band produces the shared runtime that all bands use.
+- Only 1xx and at most one other band (2xx, 3xx, or 4xx) are in support at any given
+  time.
+- The initial release of each feature band depends on the latest release of
+  the previous band.
+- 3xx and 4xx bands follow identical build patterns; the scenarios documented
+  for 3xx apply equally to 4xx with adjusted version numbers.
+
+## Build Command Arguments
+
+Feature band builds make use of these relevant command-line arguments:
+
+- `--with-packages` - Provides PSB artifacts
+- `--with-shared-components` - Provides shared component artifacts from 1xx build
+- `--with-sdk` - Specifies the SDK to use for building
+
+## Distro Maintainer Workflows
+
+The following sections describe the workflows for different scenarios.
+
+### Input Artifacts Summary
+
+**For 1xx band builds:**
+
+- **[Bootstrap](#1xx-band-bootstrap)**:
+  - PSB artifacts: Microsoft-built previous N.0.1xx release
+  - SDK: Microsoft-built previous N.0.1xx release
+- **[Servicing](#1xx-band-servicing)**:
+  - PSB artifacts: Source-built previous N.0.1xx release
+  - SDK: Source-built previous N.0.1xx release
+
+**For 2xx band builds:**
+
+- **[Initial release (N.0.200)](#2xx-band-initial-release-n0200)**:
+  - Shared component artifacts: Source-built current N.0.1xx release
+  - PSB artifacts: Source-built previous N.0.1xx release
+  - SDK: Source-built previous N.0.1xx release
+- **[Bootstrap (N.0.200+)](#2xx3xx4xx-bootstrap-n0b00)**:
```

Comment:
Consider listing this last just like it is done in the detailed section.  It is an edge case scenario.